spk/owncloud/src/app/config

```diff
@@ -1,15 +1,14 @@
 {
-    ".url": {
-        "com.synocommunity.packages.owncloud": {
-            "title": "ownCloud",
-            "desc": "ownCloud",
-            "icon": "images/owncloud-{0}.png",
-            "type": "url",
-            "protocol": "http",
-            "port": "80",
-            "url": "/owncloud",
-            "allUsers": true,
-            "grantPrivilege": "local"
-        }
-    }
+	".url": {
+		"com.synocommunity.packages.owncloud": {
+			"allUsers": true,
+			"desc": "ownCloud",
+			"icon": "images/owncloud-{0}.png",
+			"port": "443",
+			"protocol": "https",
```

Comment:
Doesn't this bit depend on local configuration ?

While the http may always work I'm not sure the https would always be working, right ?

Reply:
This shouldn't be in there at all since in the developer guide there are no attributes for `port` or `protocol` for the config file. I found however that not putting them in causes the browser to go to the DSM port (:5000 or :5001) when using DSM 6. So I put these in to get it to behave. You are right they could also be `80` and `http` but it doesn't seem to make a difference as it does http anyway and I redirect to https in the Apache conf file. I can change it if you feel strongly about it.

Reply:
TBH I don't have any strong feeling about it, I was only worried about users who wouldn't have configured https on their Synology could see odd behaviour. If it does the trick, fine by me ;)

Reply:
Okay, I'll leave as is then. My other motivation was to make it consistent with this part of the Makefile:

https://github.com/SynoCommunity/spksrc/blob/d1d3dfda78b5e170724a9712b22a19e05a0cfead/spk/owncloud/Makefile#L27-L29